src/main/cpp/t04_both2.cpp

```diff
@@ -23,9 +23,30 @@
 
 #include "t04_both2.h"
 #include <iostream>
+#include <set>
 
 using namespace std;
 
 int t04_both2() {
-
+	set <int> mif, leg, kap;
+	int cry, bae;
+	cin >> cry;
+	for (int i = 0; i < cry; i++)
+	{
+		int tim;
+		cin >> tim;
+		mif.insert(tim);
+	}
+	cin >> bae;
+	for (int i = 0; i < bae; i++)
+	{
+		int lou;
+		cin >> lou;
+		leg.insert(lou);
+		if (mif.find(lou) != mif.end())
+			kap.insert(lou);
+	}
+	set <int>::iterator j;
+	for (j = kap.begin(); j != kap.end(); j++)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Prefer prefix ++/-- operators for non-primitive types.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4678692)